app/views/categories/index.html.erb

```diff
@@ -15,7 +15,8 @@
           <%= link_to category.name, "#category#{category.id}", role: :button,
           data: { parent: '#accordion', toggle: :collapse },
           aria: { controls: "icon-indicator", expanded: true } %>
-          <span class="pull-right"><%= link_to "add section", category_path(category) %></span>
+          <% if current_user.admin %>
+          <span class="pull-right"><%= link_to "add section", category_path(category) %></span><% end %>
```

Comment:
Can we make the `end` on a separate line here and work on the indentation too.  Should be like this:

```
<% if current_user.admin %>
    <span class="pull-right"><%= link_to "add section", category_path(category) %></span>
<% end %>
```